packages/models/opportunity.ts

```diff
@@ -6,8 +6,9 @@ export interface Opportunity {
   id: string;
   attributes: {
     title: string;
-    //skills: Skill[];
+    skills: Skill[];
```

Comment:
I'm not seeing this field in the backend yet, so can we leave this commented out here?